mbof/models.py

```diff
@@ -144,9 +144,10 @@ class Intention(models.Model):
     INTENTION_DECLINED = 'declined' #may use in future
     event = models.ForeignKey(Event)
     respondent = models.ForeignKey(User, default=currentUserObject, editable=False)
-    intention = models.CharField(max_length=5, choices=(
+    intention = models.CharField(max_length=8, choices=(
```

Comment:
Yes, a max length must be defined. Otherwise:

vagrant@vagrant-ubuntu-trusty-64:/vagrant$ python manage.py makemigrations
SystemCheckError: System check identified some issues:

ERRORS:
mbof.Intention.intention: (fields.E120) CharFields must define a 'max_length' attribute.